adr/decisions/04-28-2025-default-kas-keys.md

```diff
@@ -0,0 +1,159 @@
+---
+# Required
+status: 'proposed'
+date: '2025-04-28'
```

Comment:
Recommend renaming this file , to `adr/decisions/2025-04-28-default-kas-keys.md` to match this date string and to simplify sorting (1 year from now, e.g. when using `ls -lart` in the decisions directory).